msteams-platform/TOC.yml

```diff
@@ -360,24 +360,26 @@
   - name: Prerequisites
     href: tabs/how-to/tab-requirements.md
     displayName: clickjacking
-  - name: Create a personal tab
+  - name: Create a tab
```

Comment:
Can we make sure the TOC calls this section "Create a tab"?
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/83943/502466ce-b19f-4c32-bcbf-af910568b3c2)
We should have one section for creating a tab, whether that's a personal tab or a channel tab. So we shouldn't have two pages (one for personal and one for channels) -just one "create a tab". That page should show you how to create a tab using static tabs, and how you can make them work in a personal scope and other scopes (chats, channel, meetings).

Reply:
Ah, looks like I was viewing an outdate version. This is much better
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/83943/33531f6a-1275-48b2-95ae-9d52f19344fe)